lochness/__init__.py

```diff
@@ -88,15 +88,15 @@ def initialize_metadata(Lochness, args) -> None:
     for study_name in args.studies:
         # if 'redcap' or 'rpms' is in the sources, create (overwrite)
         if 'redcap' in args.input_sources:
-            id_fieldname = 'record_id1'
-            consent_fieldname = 'Consent'
+            id_fieldname = 'chric_subject_id'
```

Comment:
This was already done in PR https://github.com/AMP-SCZ/lochness/pull/27/files. Why is this PR again showing this change?

Reply:
Hmm not sure. I must have made mistake somewhere. I'll make sure I keep these lines.